model/host/db.go

```diff
@@ -334,6 +334,15 @@ func ByRunningWithTimedOutLCT(currentTime time.Time) db.Q {
 	})
 }
 
+func RemoveAllInitializing() error {
+	return db.Remove(
```

Comment:
this should be ``db.RemoveAll``, which I got scared off of after an earlier botched attempt at this where I confused StartedBy and UserHost